core/src/main/java/org/apache/calcite/sql/type/OperandTypes.java

```diff
@@ -240,6 +240,9 @@ public Consistency getConsistency() {
   public static final SqlSingleOperandTypeChecker INTERVAL =
       family(SqlTypeFamily.DATETIME_INTERVAL);
 
+  public static final SqlSingleOperandTypeChecker STRING_STRING_DATETIME =
+      family(SqlTypeFamily.STRING, SqlTypeFamily.STRING, SqlTypeFamily.DATETIME);
+
```

Comment:
`STRING` or `CHARACTER` ? `STRING ` contains type name `BINARY` and `VARBINARY`.

Reply:
it _looks_ like Redshift doesn't have binary, so I'll change to character 👍